packages/contentful/migrations/gp2/contributingCohortsMembership/20231016142409-remove-study-link.js

```diff
@@ -0,0 +1,39 @@
+module.exports.description = 'removes study link';
+
+module.exports.up = (migration) => {
+  const contributingCohorts = migration.editContentType(
+    'contributingCohortsMembership',
+  );
+  contributingCohorts.deleteField('studyLink');
```

Comment:
don't we need to migrate the data before deleting this?

Reply:
Yes, it's been migrated.